README-TA.md

```diff
@@ -132,10 +132,9 @@ GitHub இல் பங்களிக்க பொருத்தமான ச
 
 ## திறமூல பங்களிப்பு முயற்சிகள்
 - [Up For Grabs](https://up-for-grabs.net/) - Contains projects with beginner-friendly issues
-- [First Timers Only](https://www.firsttimersonly.com/) - A list of bugs that are labelled "first-timers-only".
```

Comment:
This can be kept and added back in.

Reply:
https://github.com/freeCodeCamp/how-to-contribute-to-open-source/pull/826#discussion_r1027057240

Reply:
add66b602f87077fb9a28bfc9c24e09203be26b5 done